lib/stdlib/test/erl_lint_SUITE.erl

```diff
@@ -4950,7 +4950,24 @@ inline_nifs(Config) ->
               gurka() -> ok.
              ">>,
            [],
-           {warnings,[{{2,22},erl_lint,nif_inline}]}}],
+           {warnings,[{{2,22},erl_lint,nif_inline}]}},
+           {explicit_inline_unloaded,
+           <<"-compile({inline, [bar/0]}).
+              -nifs([bar/0]).
```

Comment:
I think there is a testing case missing :)
The case were the nif loads a single function
```erlang
-nifs(bar/0).
```